tools/dcos_login.py

```diff
@@ -203,16 +174,80 @@ def ignore_empty(envvar, default):
     dcos_login_username = ignore_empty("DCOS_LOGIN_USERNAME", __CLI_LOGIN_EE_USERNAME)
```

Comment:
If this was called `DCOS_USERNAME` it'd be picked up automatically by `dcos cluster setup` without the need to pass a flag. The password can be read from `DCOS_PASSWORD`.

Reply:
Is this the behavior for 1.9 cli as well (https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.9/dcos) ?

Reply:
No it isn't, this is something new.

Reply:
IMHO, I'd not support 1.9 here. Are we not dropping support with 1.12 for 1.9 anyways?

Reply:
Yes, I have removed this code now. But we still support 1.9 for sdk-40 branch (at least until 1.12 GA), so this is still relevant there.